Dockerfile

```
@@ -1,26 +1,51 @@
+ARG VERSION=unspecified
+
+FROM python:3.6-slim-buster
+
+ARG VERSION
```

I don't believe this is necessary in this project.

```suggestion
FROM python:3.6-slim-buster
```